src/util/newconfig/ObjectView.cpp

```diff
@@ -44,50 +44,59 @@ ObjectView::ObjectView(std::string_view prefix, std::size_t arrayIndex, ClioConf
 {
 }
 
-std::string
-ObjectView::getFullKey(std::string_view key) const
-{
-    return fmt::format("{}.{}", prefix_, key);
-}
-
 bool
 ObjectView::containsKey(std::string_view key) const
 {
-    std::string fullKey = getFullKey(key);
-    return clioConfig_.map_.contains(fullKey);
-}
-
-bool
-ObjectView::startsWithKey(std::string_view key) const
-{
-    for (auto const& [valueKey, value] : clioConfig_.map_) {
-        if (valueKey.starts_with(key))
-            return true;
-    }
-    return false;
+    auto const fullKey = getFullKey(key);
```

Comment:
Sometimes it's not worth making a constant/variable (even though generated code will be the same). I'd just write `return clioConfig_.get().contains(getFullKey(key));` here to make this a one-liner. This is always going to be a case by case thing.